query-service/src/integrationTest/java/org/hypertrace/core/query/service/QueryServiceTestUtils.java

```diff
@@ -103,10 +113,38 @@ public static Expression createStringLiteralValueExpression(String value) {
         .build();
   }
 
-  public static Expression.Builder createComplexAttributeExpression(
-      String attributeId, String subPath) {
-    return Expression.newBuilder()
-        .setAttributeExpression(
-            AttributeExpression.newBuilder().setAttributeId(attributeId).setSubpath(subPath));
+  public static QueryRequest getAttributeExpressionQuery(QueryRequest originalQueryRequest)
+      throws InvalidProtocolBufferException {
+    // Serialize into json.
+    String json =
+        JsonFormat.printer().omittingInsignificantWhitespace().print(originalQueryRequest);
+    System.out.println(json);
+
+    // Change for attribute Expression
+    json = json.replaceAll("columnIdentifier", "attributeExpression");
```

Comment:
We don't have any existing columnIdentifier queries with map field with CONTAINS_KEY_VALUE, right? If, so, it will fail the transformation.